fuzz/fuzz.c

```diff
@@ -390,6 +390,16 @@ int main(int argc, char *argv[])
     return 0;
 }
 #elif ASS_FUZZMODE == FUZZMODE_LIBFUZZER
+int LLVMFuzzerInitialize(int *argc, char ***argv)
+{
+#ifdef ASSFUZZ_FONTCONFIG_SYSROOT
+#define STR_(x) #x
+#define STR(x) STR_(x)
+    setenv("FONTCONFIG_SYSROOT", STR(ASSFUZZ_FONTCONFIG_SYSROOT), 1);
```

Comment:
this should also be set for `FUZZMODE_AFLXX_SHAREDMEM` before entering the sample loop